### ISSUE TITLE
Getting Started with Elm

### ISSUE THREAD

**Issue by spadin** [elm-lang/elm-lang.org#532]
The Elm documentation on syntax, style and features is great, but I can't find information on how to compile and run the code. Am I missing something?

**Comment by jvoigtlaender** [comment]
http://elm-lang.org/get-started#tools

**Comment by spadin** [comment]
Thank you! Is this page linked to somewhere on the site?

**Comment by jvoigtlaender** [comment]
Sadly, not at enough places! It is linked to from http://elm-lang.org/install, but I have been lobbying for linking to it from other places at well. For example, https://github.com/elm-lang/elm-lang.org/pull/379#issuecomment-165380111.

**Comment by spadin** [comment]
Thank you. I found it on the install page myself after cloning this repo and grepping for `get-started` :smile: I agree with you, it'd be nice if that link was more prominent or linked to from several places. I downloaded the Elm installer almost a month ago but haven't played with Elm yet because I didn't know where to go from there (and a little bit of laziness on my part as well). Looking forward to hack a little on Elm this weekend. Thanks again!
